lib/dialects/postgres/query-generator.js

```diff
@@ -375,7 +375,7 @@ var QueryGenerator = {
 
     var replacements = {
       table: this.quoteIdentifiers(tableName),
-      where: this.getWhereConditions(where),
+      where: this.getWhereConditions(where, null, model, options),
```

Comment:
whereItemsQuery in the same file, getWhereConditions mostly proxies to that, i'm not sure if it actually does anything anymore.